src/routes/authMiddleWare.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { Navigate, Outlet } from 'react-router';
+import PropTypes from 'prop-types';
+import TokenManager from '../utils/tokenManger';
+
+const AuthMiddleWare = ({ children, layout: Layout, isAuthProtected }) => {
+  console.log(isAuthProtected);
+
+  console.log(TokenManager.hasToken());
```

Comment:
- [ ] Kindly remove this console statements in order to improve the cleanliness of your app.